netbox/dcim/models/__init__.py

```diff
@@ -435,9 +435,20 @@ def _draw_empty(drawing, rack, start, end, text, id_, face_id, class_, reservati
 
     def _draw_elevations(self, elevation, reserved_units, face, unit_width, unit_height):
 
-        drawing = self._setup_drawing(unit_width, unit_height * self.u_height)
+        # Setup vars for the rack unit
+        position_width = 30
```

Comment:
We should probably make this a keyword argument on `_draw_elevations()` to ensure flexibility. We can set it to 30 as a default, but it should be defined as a constant like we do with `RACK_ELEVATION_UNIT_WIDTH_DEFAULT` and `RACK_ELEVATION_UNIT_HEIGHT_DEFAULT`.

Also, what do you think about calling this something like `legend_width` instead? Position seems a bit confusing given the context.

Reply:
Yeah, I think legend might be better.

Will do the constants ASAP